docs/wiki/TestingDesign.md

```diff
@@ -113,8 +113,8 @@ Since also dependency resources are in turn subject to dependencies with each ot
 
 **Second level resources**: This group of resources has a dependency only on the resource group which will host them. Resources in this group can be deployed in parallel.
 
-  1. User assigned identity: This resource is leveraged by all dependency resources
-      > **Note**: The object ID of the [user assigned identity] must be set in several dependency parameter files. However, when you first run the pipeline, this object ID will be unknown. It is hence recommended to either manually create the MSI beforehand - or - run the pipeline without the ID once (which will cause the pipeline to fail during the ID's usage, but **after** the MSI was deployed), then update the value in the parameter files and finally re-run the pipeline.
+  1. User assigned identity: This resource is leveraged by the [role assignment], [key vault] and [recovery services vault] dependency resources.
+      > **Note**: The object ID of the [user assigned identity] is needed by several dependency parameter files. However, before running the dependency pipeline for the first time, the [user assigned identity] resource does not exist yet, thus its object ID is unknown. For this reason, instead of the object ID value, some dependency parameter files contain the `"<<msiPrincipalId>>"` token, for which the correct value is retrieved and replaced by the pipeline at runtime.
```

Comment:
I'd almost be tempted to say - let's not talk about it anyways. But at the same time it may make sense to explain the MSI token afterall. So this comment is just a 'soft' challenge :D

Reply:
I thought the same but since we had this note before my instinct was updating it instead of deleting it. I'm good with either

Reply:
We anyway mention that already in the parameter file (Replaced by the pipeline) and that's for now only used by the dependency pipeline